csv/src/main/java/org/motechproject/nms/csv/utils/CsvInstanceImporter.java

```diff
@@ -17,14 +30,124 @@ public CsvInstanceImporter(Class<T> type) {
 
     public T read() throws IOException {
         if (isOpen()) {
-            return getCsvReader().read(type, getFieldNames(), getProcessors());
+            T instance = createInstance();
+            Map<String, String> cells = csvReader.read(columnNames);
+            if (null != cells) {
```

Comment:
We've moved away from doing `null != value` comparisons since java won't allow non bool assignments in if clauses 
and our checkstyle rules break anyway.